Data/future/cme/margins/ESK.csv

```diff
@@ -1,40 +1,42 @@
 date,initial,maintenance
-# Values are converted from SEK to USD
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv12-270.html
 20120629,2341,1734
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv12-380.html
 20120907,2335,2123
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv13-069.html
+20130102,15400,14000
 20130208,2048,1862
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv13-188.html
+20130211,13200,12000
 20130423,2317,2106
```

Comment:
Values are in the wrong order of magnitude. It's probably in another currency (Swedish Krona maybe?)